Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Build #
+FROM node:15-buster as build
+WORKDIR /usr/app
+
+COPY package.json package-lock.json ./
+RUN npm install
+
+COPY src ./src/
```

Comment:
MONGO_URI is not setted on docker as an ENV. I'm not sure it would work.

Reply:
You don't need to set env variables at dockerfile. Dockerfiles are used to create image and images should not includes any env.
Developer can pass environments to container at the runtime, `--env` or `--env-file`.
There should be an env file to describe what project needs. @cagataycali may set it on the project.